utils/wasm-builder/src/wasm_project.rs

```diff
@@ -127,7 +127,7 @@ impl WasmProject {
             .expect("Could not find target directory");
 
         let mut wasm_target_dir = target_dir.clone();
-        wasm_target_dir.push("wasm32-unknown-unknown");
+        wasm_target_dir.push("gear-wasm");
```

Comment:
```suggestion
        wasm_target_dir.push("wasm32-gear");
```

Reply:
how did you pass CI? 🤔

Reply:
ok, let's run wasm builder smoke tests on CI: https://github.com/gear-tech/gear/actions/runs/12823207597